src/main/scala/com/cognite/sdk/scala/v1/files.scala

```diff
@@ -94,15 +99,19 @@ final case class FileUpdate(
     sourceCreatedTime: Option[Setter[Instant]] = None,
     sourceModifiedTime: Option[Setter[Instant]] = None,
     securityCategories: Option[NonNullableSetter[Seq[Long]]] = None,
-    dataSetId: Option[Setter[Long]] = None
+    dataSetId: Option[Setter[Long]] = None,
+    labels: Option[NonNullableSetter[Seq[CogniteExternalId]]] = None
```

Comment:
this, I am again not sure. This allows `set` operation on labels but files doesn't support that.

Reply:
I think that it supports it: https://docs.cognite.com/api/v1/#operation/updateFiles

![image](https://user-images.githubusercontent.com/7894687/146168763-06f69000-696d-4be0-9c54-a5b56067caff.png)

Reply:
it wasn't there when i checked 😱 maybe sth new?

Reply:
strange, I think I missed it, sorry for blocking the PR